packages/design-system-tailwind-preset/jest.config.js

```diff
@@ -3,14 +3,16 @@
  * https://jestjs.io/docs/configuration
  */
 
-const merge = require('deepmerge');
 const path = require('path');
 
 const baseConfig = require('../../jest.config.packages');
 
 const displayName = path.basename(__dirname);
 
-module.exports = merge(baseConfig, {
```

Comment:
Merge will merge the array of coveragePathIgnorePatterns instead of replacing it. So removing here for simple spread configuration.